doc/markup/code.rst

```diff
@@ -182,13 +182,24 @@ Includes
    ``start-after`` is given as a string option, only lines that follow the first
    line containing that string are included.  If ``end-before`` is given as a
    string option, only lines that precede the first lines containing that string
-   are included. If used in combination with ``lines``, make sure the latter
-   allows the lines (whose count start at ``1`` at top of file) containing the
-   looked-for strings.
+   are included.
+
+   With lines selected this way it is still possible to use ``lines``, the
+   numbers being now interpreted relative to the already selected lines.
+
+   .. versionchanged:: 1.5.3
+      Formerly, ``lines`` still referred to the included file, and it was
+      needed that those lines containing the requested strings were among
+      the allowed ones.
+
```

Comment:
perhaps it would be better to suppress this from the docs, because in fact using both of lines and start-after earlier was contra-intuitive, and it is perhaps better not to include an a posteriori explanation which was absent of docs actually